_pytest/_code/source.py

```diff
@@ -52,6 +52,9 @@ def __eq__(self, other):
                 return str(self) == other
             return False
 
+    def __hash__(self):
```

Comment:
@RonnyPfannschmidt @The-Compiler What would you suggest is a better hashable value?